### ISSUE TITLE
Visual style for "Commit Selected" broken somehow

### ISSUE THREAD

**Issue by juliohm1978** [JohnstonCode/svn-scm#749]
- VSCode Version: 1.40.1
- OS Version: Ubuntu 18.04 LTS
- Extension Version: 2.1.3
- System Language: English
- SVN Version: 1.9.7

## Issue ##

With so many extensions installed, eventually I started seeing this.

![image](https://user-images.githubusercontent.com/4145456/69670155-fff7cc00-1071-11ea-8f04-031bb298dc30.png)

The "Commit Selected" page seems to have its css style messed up.

## Steps to Reproduce ##

Unsure how to reproduce. Seems related to my local `.vscode`. Could be corrupted?

Looking for tips to fix this without rebuilding my entire `.vscode` directory.

Thanks in advance.

**Comment by juliohm1978** [comment]
PS: I did try to remove several extensions I haven't used much. Most themes and icons packs.

SVN extension was also remove and reinstalled. The problem persists.

**Comment by JohnstonCode** [comment]
Do you have any errors in the developer console? `Help >> Toggle Developer tools >> console`

**Comment by edgardmessias** [comment]
Looking the image, I see the problem is the CSS is not loaded.
I guess the problem is the relative URL to CSS file

**Comment by juliohm1978** [comment]
That's right. The debug console shows this error.

![image](https://user-images.githubusercontent.com/4145456/69879650-c0fe8c00-12a6-11ea-9e52-2f9ab2470fd3.png)

Not sure how to resolve this, though.

**Comment by juliohm1978** [comment]
This might give a better insight into the error.

![image](https://user-images.githubusercontent.com/4145456/69879922-6d407280-12a7-11ea-9ac1-aba934eaa288.png)

Relates to:

```javascript
if (result.type === 'access-denied') {
  console.error('Webview: Cannot load resource outside of protocol root');
  return callback({ error: -10 /* ACCESS_DENIED: https://cs.chromium.org/chromium/src/net/base/net_error_list.h */ });
}
```

**Comment by edgardmessias** [comment]
@juliohm1978 , call me in https://gitter.im/svn-scm/Lobby

Try open the file `.../.vscode/extensions/johnstoncode.svn-scm-2.1.3/out/messages.js` and edit:
Add after `<body>` the `${panel.webview.cspSource}` and `${styleUri}`, to show the URLs

**Comment by juliohm1978** [comment]
Sorry for the late response. Had busy week.

I managed to edit the `messages.js` file to show these values and reload vscode.

```
panel.webview.cspSource: vscode-resource:

styleUri: vscode-resource://file///home/lamento/workspace/gitlab/dotfiles/trabalho/home/lamento/.vscode/extensions/johnstoncode.svn-scm-2.1.3/css/commit-message.css
```
![image](https://user-images.githubusercontent.com/4145456/70337290-83ab7880-1829-11ea-91fd-d53ba7eba227.png)

**Comment by juliohm1978** [comment]
Noted, the file `/home/lamento/workspace/gitlab/dotfiles/trabalho/home/lamento/.vscode/extensions/johnstoncode.svn-scm-2.1.3/css/` is present and accessible by my user.

```
# cd /home/lamento/workspace/gitlab/dotfiles/trabalho/home/lamento/.vscode/extensions/johnstoncode.svn-scm-2.1.3/css/

# ls -l 
total 20K
-rw-r--r-- 1 lamento lamento 8,5K Nov 16 15:32 commit-message.css

**Comment by juliohm1978** [comment]
Ok, so I managed to figure out the reason why it's not working!

Showing the values on screen, I realized that my `.vscode` is not in the original user home directory. I recently had it moved to `/home/lamento/workspace/gitlab/dotfiles/trabalho/home/lamento/.vscode` and created a symlink to it. My intention was to commit my `.vscode` directory to a git repository as a backup.

Moving `.vscode` back to my home directory fixed the issue.

Thank you!
